### ISSUE TITLE
v0.4.7-libbpf-1.2.0-b2e29a1 tag is released, but libblf v1.2.0 does not exist

### ISSUE THREAD

**Issue by bobrik** [aquasecurity/libbpfgo#315]
See: https://github.com/aquasecurity/libbpfgo/releases/tag/v0.4.7-libbpf-1.2.0-b2e29a1

<img width="824" alt="image" src="https://user-images.githubusercontent.com/89186/229594570-d50dd430-66d3-477f-bd8d-3a5a355a6287.png">

The title of the release says libbpf-1.1.0, but it's libbpf-1.2.0 in the tag. The commit hash also doesn't match.

I received a dependabot PR for this as well:

* https://github.com/cloudflare/ebpf_exporter/pull/198

**Comment by rafaeldtinoco** [comment]
Done @bobrik. Thank you! @geyslan make sure to delete your local tags (for the next pushes). Cheers!

**Comment by bobrik** [comment]
The changelog link needs adjustment here:

* https://github.com/aquasecurity/libbpfgo/releases/tag/v0.4.7-libbpf-1.1.0-41-gb2e29a1

> Full Changelog: https://github.com/aquasecurity/libbpfgo/compare/v0.4.6-libbpf-1.1.0...v0.4.7-libbpf-1.2.0-b2e29a1

**Comment by bobrik** [comment]
The latest release tag also looks very different from the previous ones, not sure if that's intentional:

<img width="320" alt="image" src="https://user-images.githubusercontent.com/89186/229626166-247fdbb4-01a4-4538-8b56-d40a421ddb24.png">

**Comment by rafaeldtinoco** [comment]
> The latest release tag also looks very different from the previous ones, not sure if that's intentional:

yes, it is. we have bumped libbpf to a specific commit (before a release was made) due to that fix. best world would be to bump libbpf to v1.2.0 but that has not been released yet and we needed the fix.

thanks again!
